staging/src/k8s.io/legacy-cloud-providers/azure/azure.go

```diff
@@ -544,10 +544,10 @@ func (az *Cloud) configAzureClients(
 	diskClientConfig := azClientConfig.WithRateLimiter(az.Config.DiskRateLimit)
 	vmClientConfig := azClientConfig.WithRateLimiter(az.Config.VirtualMachineRateLimit)
 	vmssClientConfig := azClientConfig.WithRateLimiter(az.Config.VirtualMachineScaleSetRateLimit)
-	// Error "not an active Virtual Machine Scale Set VM" is not retriable for VMSS VM.
-	// But http.StatusNotFound is retriable because of ARM replication latency.
```

Comment:
both retriable and retryable are correct words, I prefer retriable (similar format as triable)